gateway/src/main/kotlin/com/makarytskyi/gateway/handler/GlobalExceptionHandler.kt

```diff
@@ -1,5 +1,6 @@
-package com.makarytskyi.rentcar.exception
+package com.makarytskyi.gateway.handler
```

Comment:
It is great that you have global exception hanler for `gateway`, but who will handle excpetions for `domainservice`?
I mean you still have CarController, RepairingController and UserController